src/core/urls.py

```diff
@@ -4,6 +4,7 @@
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 from django.views.generic import TemplateView
 
+import views
```

Comment:
I am also a supporter of importing all the views at the top and have smaller / more readable code below. Also, importing all the views at the top would make it easy to spot what is currently used in the file, when refactoring.